spec/facades/search_facade_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe SearchFacade do
+  before(:each) do
+    @current_user = User.create!(first_name: "Chicken", last_name: "Bird", email: "chicken@gmail.com", street: "1234 Street", city: "CO springs", state: "CO", zipcode: "12345", lat: 1.12, lon: 1.12, is_remote: false, about: "I enjoy long walks on the beach")
+    @found_user = User.create!(first_name: "Kiwi", last_name: "Bird", email: "kiwi@gmail.com", street: "1234 Street", city: "CO springs", state: "CO", zipcode: "12345", lat: 1.25, lon: 1.32, is_remote: false, about: "I enjoy long walks on the beach")
+    @found_user_skill = Skill.create!(name: "Testing", proficiency: 5, user_id: @found_user.id)
+
+    @params = {
+      is_remote: false,
+      user_id: @current_user.id,
+      query: "testing"
+    }
+  end
+
+  it "exists and has unreadable attributes on initialize" do
+    facade = SearchFacade.new(@params)
+    expect(facade).to be_a SearchFacade
+    expect {facade.is_remote}.to raise_error(NoMethodError)
```

Comment:
I'm curious as to why this would be raising a nomethod error. Aren't these populating the attributes? Should they not be readable?

Reply:
I'm thinking it's because it has to build the users first using the build users method so that it can be called on those users

Reply:
Yeah, this is just extraneous testing really. None of the attributes are readable as they don't need to be called on anywhere outside of the class, so this was just a "confirmation of non-readability" test. Since they aren't readable, they'd raise a NoMethod exception when called.

Super extra, totally not needed in the grand scheme of things.